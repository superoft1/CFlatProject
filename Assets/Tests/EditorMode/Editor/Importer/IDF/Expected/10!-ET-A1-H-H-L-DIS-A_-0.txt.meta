fileFormatVersion: 2
guid: 61a3b0525581e2c44aea728a31e6268b
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
