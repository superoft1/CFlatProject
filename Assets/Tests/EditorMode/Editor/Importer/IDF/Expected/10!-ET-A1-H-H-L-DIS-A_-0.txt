WeldNeckFlange, , (0.0, 0.0, -1.0), (-101.7, 28.0, 1.3), False
(-101.7, 28.0, 1.2)
(-101.7, 28.0, 1.3)
FreeVertex: 1
Pipe, , (0.0, 0.0, 1.0), (-101.7, 28.0, 1.4), False
(-101.7, 28.0, 1.3)
(-101.7, 28.0, 1.5)
FreeVertex: 0
ConcentricPipingReducer, , (0.0, 0.0, 1.0), (-101.7, 28.0, 1.6), False
(-101.7, 28.0, 1.7)
(-101.7, 28.0, 1.5)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-101.7, 28.0, 2.0), False
(-101.7, 28.0, 1.7)
(-102.1, 28.0, 2.0)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-102.3, 28.0, 2.0), False
(-102.1, 28.0, 2.0)
(-102.5, 28.0, 2.0)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, 1.0), (-102.5, 28.0, 2.0), False
(-102.5, 28.0, 2.0)
(-102.5, 28.0, 2.0)
(-102.5, 28.0, 2.2)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-102.5, 28.0, 2.2), False
(-102.5, 28.0, 2.2)
(-102.5, 28.0, 2.3)
FreeVertex: 0
GlobeValve, , (0.0, 0.0, 1.0), (-102.5, 28.0, 2.3), False
(-102.5, 28.0, 2.3)
(-102.5, 28.0, 2.3)
FreeVertex: 0
PressureGauge, PG_102_11_10_1070_1E, (0.0, 0.0, 1.0), (-102.5, 28.0, 2.4), False
(-102.5, 28.0, 2.4)
(-102.5, 28.0, 2.3)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-102.6, 28.0, 2.0), False
(-102.5, 28.0, 2.0)
(-102.7, 28.0, 2.0)
FreeVertex: 0
Elbow90, , (0.0, 0.0, 1.0), (-103.0, 28.0, 2.0), False
(-102.7, 28.0, 2.0)
(-103.0, 28.0, 1.7)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-103.0, 28.0, 1.3), False
(-103.0, 28.0, 1.7)
(-103.0, 28.0, 1.0)
FreeVertex: 0
Elbow90, , (0.0, -1.0, 0.0), (-103.0, 28.0, 0.6), False
(-103.0, 28.0, 1.0)
(-103.0, 28.4, 0.6)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-103.0, 28.5, 0.6), False
(-103.0, 28.4, 0.6)
(-103.0, 28.5, 0.6)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-103.0, 28.6, 0.6), False
(-103.0, 28.6, 0.6)
(-103.0, 28.5, 0.6)
FreeVertex: 0
CheckValve, , (0.0, 1.0, 0.0), (-103.0, 28.7, 0.6), True
(-103.0, 28.8, 0.6)
(-103.0, 28.6, 0.6)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-103.0, 28.8, 0.6), False
(-103.0, 28.8, 0.6)
(-103.0, 28.9, 0.6)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-103.0, 29.0, 0.6), False
(-103.0, 28.9, 0.6)
(-103.0, 29.1, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-103.0, 29.1, 0.6), False
(-103.0, 29.1, 0.6)
(-103.0, 29.1, 0.6)
(-103.0, 29.1, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-103.0, 29.1, 0.4), False
(-103.0, 29.1, 0.5)
(-103.0, 29.1, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-103.0, 29.1, 0.4), False
(-103.0, 29.1, 0.3)
(-103.0, 29.1, 0.4)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-103.0, 29.1, 0.6), False
(-103.0, 29.1, 0.6)
(-103.0, 29.2, 0.6)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-103.0, 29.3, 0.6), False
(-103.0, 29.3, 0.6)
(-103.0, 29.2, 0.6)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-103.0, 29.3, 0.6), False
(-103.0, 29.3, 0.6)
(-103.0, 29.3, 0.6)
FreeVertex: 0
GateValve, , (0.0, 1.0, 0.0), (-103.0, 29.5, 0.6), False
(-103.0, 29.7, 0.6)
(-103.0, 29.3, 0.6)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-103.0, 29.7, 0.6), False
(-103.0, 29.7, 0.6)
(-103.0, 29.8, 0.6)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-103.0, 29.9, 0.6), False
(-103.0, 29.8, 0.6)
(-103.0, 29.9, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-103.0, 29.9, 0.6), False
(-103.0, 29.9, 0.6)
(-103.0, 29.9, 0.6)
(-103.0, 29.9, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-103.0, 29.9, 0.4), False
(-103.0, 29.9, 0.5)
(-103.0, 29.9, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-103.0, 29.9, 0.4), False
(-103.0, 29.9, 0.3)
(-103.0, 29.9, 0.4)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-103.0, 30.7, 0.6), False
(-103.0, 29.9, 0.6)
(-103.0, 31.4, 0.6)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-103.0, 31.8, 0.6), False
(-103.0, 31.4, 0.6)
(-103.0, 31.8, 1.0)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-103.0, 31.8, 1.7), False
(-103.0, 31.8, 1.0)
(-103.0, 31.8, 2.5)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-103.0, 31.8, 2.8), False
(-103.0, 31.8, 2.5)
(-103.4, 31.8, 2.8)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-104.1, 31.8, 2.8), False
(-103.4, 31.8, 2.8)
(-104.8, 31.8, 2.8)
FreeVertex: 0
Tee, , (-1.0, 0.0, 0.0), (-105.0, 31.8, 2.8), True
(-105.2, 31.8, 2.8)
(-104.8, 31.8, 2.8)
(-105.0, 31.8, 3.1)
FreeVertex: 2
FreeVertexTotal: 6
